plugins/package-managers/cocoapods/src/main/kotlin/CocoaPods.kt

```diff
@@ -177,7 +177,7 @@ class CocoaPods(
         return Package(
             id = id,
             authors = emptySet(),
-            declaredLicenses = podspec.license.takeUnless { it.isEmpty() }?.let { setOf(it) } ?: emptySet(),
+            declaredLicenses = podspec.license.takeUnless { it.isEmpty() }?.let { setOf(it) }.orEmpty(),
```

Comment:
How about `.orEmpty().toSet()` to get rid of this `let` ?

Reply:
I don't see how that should work (`podspec.license` is a string), but I've done a commit on top that simplifies this via `setOfNotNull()`.

Reply:
Sorry, my fault. Thought podspec.license was some collection time.